mne/viz/topo.py

```diff
@@ -249,16 +249,31 @@ def _check_vlim(vlim):
 def _imshow_tfr(ax, ch_idx, tmin, tmax, vmin, vmax, onselect, ylim=None,
                 tfr=None, freq=None, vline=None, x_label=None, y_label=None,
                 colorbar=False, picker=True, cmap=('RdBu_r', True), title=None,
-                hline=None):
-    """Show time-freq map on topo."""
+                hline=None, imtype='normal'):
+    """Show time-freq map as 2d image."""
     import matplotlib.pyplot as plt
+    from matplotlib.image import NonUniformImage
     from matplotlib.widgets import RectangleSelector
 
-    extent = (tmin, tmax, freq[0], freq[-1])
     cmap, interactive_cmap = cmap
+    extent = (tmin, tmax, freq[0], freq[-1])
+
+    if imtype == 'normal':
+        img = ax.imshow(tfr[ch_idx], extent=extent, aspect="auto",
+                        origin="lower", vmin=vmin, vmax=vmax,
+                        picker=picker, cmap=cmap)
+    elif imtype == 'nonuniform':
```

Comment:
there should a test if imtype has allowable str option, I will check it if kwarg is the way to go